### PR TITLE
instructorHomePage: Show start/end date for sessions #4426

### LINKED ISSUE #4426: instructorHomePage: Show start/end date for sessions
This can be useful to instructors.

### PR DESCRIPTION
**Pull request by ashrayjain**
Fixes #4426

### CODE REVIEW

File: src/test/resources/pages/InstructorHomeHTML.html [hunk 1]
```diff
@@ -213,155 +237,173 @@ <h5 class="pull-left">
             <tbody>
                <tr id="session0">
                   <td>
-                     Second Feedback Session
+                     First Feedback Session
```
Comment (damithc) [inline src/test/resources/pages/InstructorHomeHTML.html]: How did the order of sessions change?

File: src/test/resources/pages/InstructorHomeHTML.html [hunk 2]
```diff
@@ -213,155 +237,173 @@ <h5 class="pull-left">
...
                            </a>
-                           <button aria-expanded="false" class="btn btn-default btn-xs btn-tm-actions dropdown-toggle session-remind-options-for-test" data-toggle="dropdown" type="button">
+                           <button aria-expanded="false" class="btn btn-default btn-xs btn-tm-actions dropdown-toggle session-remind-options-for-test" data-toggle="dropdown" disabled="disabled" type="button">
                               <span class="caret">
                               </span>
                            </button>
                            <ul class="dropdown-menu" role="menu">
                               <li>
-                                 <a class="session-remind-inner-for-test" href="/page/instructorFeedbackRemind?courseid=CHomeUiT.CS2104&fsname=First+Feedback+Session&user=CHomeUiT.instructor.tmms" onclick="return toggleRemindStudents('First Feedback Session');">
+                                 <a class="session-remind-inner-for-test" disabled="disabled" href="/page/instructorFeedbackRemind?courseid=CHomeUiT.CS2104&fsname=Second+Feedback+Session&user=CHomeUiT.instructor.tmms">
```
Comment (damithc) [inline src/test/resources/pages/InstructorHomeHTML.html]: This doesn't seem to be related to this PR

File: src/test/resources/pages/InstructorHomeHTML.html (hunk 1 above)
Comment (ashrayjain) [inline src/test/resources/pages/InstructorHomeHTML.html]: I added new tests that sort the sessions. To do that, I had to modify the test data bundle (src/test/resources/data/InstructorHomePageUiTest3.json). I changed the start and end dates to make the sort test meaningful. Removing these changes to the test bundle restores the order. I think changing the start and end dates caused a change in the default order of these sessions, which is why it changed here.

File: src/test/resources/pages/InstructorHomeHTML.html (hunk 2 above)
Comment (ashrayjain) [inline src/test/resources/pages/InstructorHomeHTML.html]: This is also the due to the swap. "First session" swapped with "Second session"

File: src/test/resources/pages/InstructorHomeHTML.html (hunk 1 above)
Comment (ashrayjain) [inline src/test/resources/pages/InstructorHomeHTML.html]: I verified this on master. Without any other change, just changing the start and end times in InstructorHomePageUiTest3.json causes the default order of the sessions on this page to change.

Comment (damithc) [inline src/test/resources/pages/InstructorHomeHTML.html]: Got it.

**Comment by kanghj** [comment]
@ashrayjain, merge this soon?

**Comment by ashrayjain** [comment]
Sorry for the delay! Merging it now.
